### DISCUSSION TITLE
Vscodium fails to start

### DISCUSSION THREAD

**Discussion by paulofpimenta** [dat-linux/community#72] [Issues]
Hi,

I successfully installed Vscodium (v **1.95.1.24307**)  from DatLinux Applications but the application fails to start. When I run : 

```shell
vscodium --verbose
```

I have the following message : 

```shell
[35039:1106/101615.238609:FATAL:setuid_sandbox_host.cc(163)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/..../.cari/installs/vscodium/1.95.1.24307/chrome-sandbox is owned by root and has mode 4755.
```

After doing the procedure above, the application is able to start, but it crashes when the file menu is clicked. In fact, there is an issue on vscodium git repository that was recently posted [here](https://github.com/VSCodium/vscodium/issues/2082) concerning this behaviour

Do you plan to keep this version and wait for the fix or is there another way to downgrade vscodium in the app manager ?

Thank you :-)

**Comment by dat-linux** [comment]
Hey. Thanks for the heads up. I am not seeing a problem with v1.95.1.24307 on my DAT 1.x setup. 

Are you on DAT Linux 1.x or 2.0?

Meanwhile, you can revert to a previous version:

```
cari list vscodium # get version history
cari uninstall vscodium
cari install vscodium 1.94.2.24286 # specify a version
cari global vscodium 1.94.2.24286

```

Then avoid updates until a good version arrives.

**Comment by paulofpimenta** [comment]
Hi, 

It is on a fresh install of DAT Linux 2.  I didn't have any issues on version 1.x. I will try to downgrade the version following your instructions. 

Thank you :-)

**Comment by dat-linux** [comment]
I patched the installer so it should work now for all versions. Thanks again for spotting this. (you might need to reinstall vscodium)

**Comment by paulofpimenta** [comment]
Hi,
I reinstalled vscodium and the application can start now. But the file menu bug remains. Now it is not a DAT Linux anymore :-)

**Reply by dat-linux** [reply]
Hi, yes this is a common bug for this version of VSCodium. Please revert to previous version as per the instructions above and wait for the following version release before upgrade.
